i18n/qqq.json

```diff
@@ -17,5 +17,6 @@
 	"ext-whowrotethat-tour-welcome-description": "The description for the welcome popup that appears when the extension is enabled for the first time, explaining the activation and base usage of the extension.",
 	"ext-whowrotethat-tour-welcome-dismiss": "Button to permanently dismiss the welcome popup that appears when the extension is enabled for the first time.",
 	"ext-whowrotethat-revision-added": "Message shown in a popup that indicates who authored the text of an article. $1 is the username along with (talk | contribs) links, and $2 is a link to the diff with the timestamp as the link text.",
-	"ext-whowrotethat-revision-attribution": "Message indicating how much of the page an editor authored. This is shown immediately after the {{wm-msg|ext-whowrotethat-revision-added}} message."
+	"ext-whowrotethat-revision-attribution": "Message indicating how much of the page an editor authored. This is shown immediately after the {{wm-msg|ext-whowrotethat-revision-added}} message.",
+	"ext-whowrotethat-revision-username": "The text that displays in the info bar that indicates who authored the text of an article."
```

Comment:
You should document the parameter given here; We usually add `Parameters:\n* $1 - [explanation]` at the end of the qqq message. You can see `ext-whowrotethat-error-contact` message for an example